driver-core/src/main/com/mongodb/client/model/expressions/Expressions.java

```diff
@@ -139,6 +171,14 @@ public static NumberExpression of(final Decimal128 of) {
         return new MqlExpression<>((codecRegistry) -> new AstPlaceholder(new BsonDecimal128(of)));
     }
 
+    /**
+     * Returns an {@linkplain ArrayExpression array} of
+     * {@linkplain NumberExpression numbers} corresponding to
+     * the provided {@link Decimal128 Decimal128s}.
```

Comment:
[optional]

`{@link Decimal128 Decimal128s}` -> `{@link Decimal128}s` (the same with strings and instants below)?

1. It never makes sense to duplicate the linked element name, unless you link to a fully-qualified name but want to display only the simple name.
2. "s" is not part of the type name.